.github/workflows/ci.yaml

```diff
@@ -4,6 +4,7 @@ env:
   IMAGE_NAME: tubesync
 
 on:
+  workflow_dispatch:
```

Comment:
What's the purpose of this?

Reply:
Fair enough, I just generally use `make test` locally. The CI tests were to confirm it was working before I build a release image. Providing this doesn't break anything it's fine to leave it.

Honestly, I don't use GitHub actions much so this wasn't a critique I didn't actually know what this did.